### PR TITLE
OSIM: CVSS Rescoring Workflow (Email NIST)

### PR DESCRIPTION
**Pull request by tboyetteredhat**

### CODE REVIEW

**Comment by d10n** [comment]
Approved pending test update

File: src/components/CvssNISTForm.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+Red Hat's CVSS Justification:
+____
+
+Thank you,
+-${userStore.userEmail}
+`);
+
+function openMailto() {
+  const recipient = encodeURI('nvd@nist.gov');
```
Comment (superbuggy) [inline src/components/CvssNISTForm.vue:45]: use `toEmail` here

File: src/components/CvssNISTForm.vue [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  const cc = encodeURI('secalert@redhat.com');
```
Comment (superbuggy) [inline src/components/CvssNISTForm.vue:46]: use `ccEmail` here

**Review by superbuggy — CHANGES_REQUESTED** [review]
This looks great, the tests are really thorough! Just 2 very minor changes requested

File: src/components/CvssNISTForm.vue (hunk 1 above)
Comment (tboyetteredhat) [reply src/components/CvssNISTForm.vue]: updated

File: src/components/CvssNISTForm.vue (hunk 2 above)
Comment (tboyetteredhat) [reply src/components/CvssNISTForm.vue]: updated

**Pull request merged**